kolibri/content/api.py

```diff
@@ -27,7 +27,7 @@ class ChannelMetadataFilter(filters.FilterSet):
     available = filters.django_filters.MethodFilter()
 
     def filter_available(self, queryset, value):
-        if value == "True":
+        if value == "true":
```

Comment:
I really don't like this. Is there a way we can have the frontend just return a boolean instead @indirectlylit @rtibbles?

Reply:
see also https://github.com/learningequality/kolibri/pull/2603#discussion_r151262022 :)

Reply:
No @aronasorman - because you can't encode a boolean in a GET parameter. I really wish we could!